test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -50,7 +52,9 @@ namespace integration_framework {
       milliseconds block_waiting,
       milliseconds tx_response_waiting)
       : iroha_instance_(std::make_shared<IrohaInstance>(
-            mst_support, block_store_path, dbname)),
+            mst_support, block_store_path, kToriiPort, dbname)),
+        command_client_("127.0.0.1", kToriiPort),
```

Comment:
It might be a good idea to define a `kListenIp` constant similar to `kToriiPort`

Reply:
This change and even more like this is done in a PR i am going to have ready soon. Hope you do not mind postponing this.